jkamz/Questioner
# Ft sign in 162982269

### What does this PR do?
This PR creates the sign in page for both users and admins. Admins need to check the 'sign in as admin' checkbox

### Description of the tasks to be completed
The tasks for this PR include:

- Create the sign in form
- Create a sign up link for unregistered users
- Create a checkbox for admin users

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/Questioner`

`$ cd Questioner`

Check out to the PR's branch
`$ git checkout ft-sign-in-`162982269

Navigate to UI/templates folder and open signin.html on browser

### What are the relevant Pivotal Tracker stories?
#162982269

### Screenshots
![signin](https://user-images.githubusercontent.com/11965186/50737288-6d8bd700-11d8-11e9-93dd-cef6384e426e.png)

Mobile Site

![signin-mobile](https://user-images.githubusercontent.com/11965186/50737292-7a102f80-11d8-11e9-9edf-0035a274e8de.png)
